db/seeds.rb

```diff
@@ -40,13 +46,16 @@
 		password: "password",
 		password_confirmation: "password"
 	)
-	print "❤️"; print " "
+	person = people.sample
+	print " #{person} "; print " "
```

Comment:
Just want to point out that you can do `people.sample` instead of having a valuable do this. Since you can have chain methods inside a string interpolation.